spec/models/category_spec.rb

```diff
@@ -1,5 +1,33 @@
 require 'spec_helper'
 
 describe Category do 
-  it { should have_many (:videos)}
-end
\ No newline at end of file
+  it { should have_many (:videos) }
+end
+
+describe "#recent_videos" do
+  it "returns 6 videos when there are more than 6 videos in total" do
+    comedies = Category.create(name: "comedies")
+    futurama = Video.create(title: "Futurama", description: "Space Travel", category: comedies)
+    southpark = Video.create(title: "SouthPark", description: "Crazy animated show", category: comedies)
+    back_to_future = Video.create(title: "Back to Future", description: "Time Travel", category: comedies)
+    futurama = Video.create(title: "Futurama", description: "Space Travel", category: comedies)
+    southpark = Video.create(title: "SouthPark", description: "Crazy animated show", category: comedies)
+    back_to_future = Video.create(title: "Back to Future", description: "Time Travel", category: comedies)
+    futurama = Video.create(title: "Futurama", description: "Space Travel", category: comedies)
```

Comment:
There isn't a point to store all videos as variables. You are not using them anyway.